pkg/sql/execinfrapb/processors_changefeeds.proto

```diff
@@ -97,4 +98,7 @@ message ChangeFrontierSpec {
 
   // Description is the description of the changefeed. Used for structured logging.
   optional string description = 5 [(gogoproto.nullable) = false];
+
+  // Change frontier span level checkpoint
```

Comment:
nit: comment should begin with `SpanLevelCheckpoint ...` and end with punctuation